From 20d4087669a09410ed1fd3bfaf8d295a18609fef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 7 May 2013 14:13:04 -0400
Subject: [PATCH] build-sys: properly mkdir with DESTDIR set
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

grawity> ln: failed to create symbolic link
‘/home/grawity/pkg/aur/systemd-git/pkg/systemd//etc/systemd/system/multi-user.target.wants/remote-fs.target’: No such file or directory
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 96f80d7..1b8ed92 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -244,7 +244,7 @@ install-aliases-hook:
 		dir= && $(install-aliases)
 
 define install-aliases
-	$(MKDIR_P) /$(DESTDIR)$$dir && \
+	$(MKDIR_P) $(DESTDIR)$$dir && \
 	while [ -n "$$1" ]; do \
 		rm -f $(DESTDIR)$$dir/$$2 && \
 		$(LN_S) $$1 $(DESTDIR)$$dir/$$2 && \
-- 
1.8.2.2

